src/components/icon/icons/finance.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Finance({
```

Comment:
I would try to name this for the object instead of what *we* believe is the meaning of it. Maybe `Bank` or something similar. What do you think?